extension/js/removeNestedQuotes.js

```diff
@@ -33,3 +33,62 @@ if (onlyLastQuote) {
 		messageArea.innerHTML = text;
 	}
 }
+
+function lastaddquote(post_id, username, l_wrote) {
+	var message_name = 'message_' + post_id;
+	var theSelection = '';
+	var divarea = false;
+	var i;
+
+	if (l_wrote === undefined) {
+		// Backwards compatibility
+		l_wrote = 'wrote';
+	}
+
+	if (document.all) {
+		divarea = document.all[message_name];
+	} else {
```

Comment:
why do you need this? why not just use `getElementById`?
`document.all` is Deprecated